beetsplug/spotify.py

```diff
@@ -164,6 +164,12 @@ def _handle_response(self, request_type, url, params=None):
                 )
                 self._authenticate()
                 return self._handle_response(request_type, url, params=params)
+            elif response.status_code == 429:
+                seconds = response.headers['Retry-After']
+                time.sleep(int(seconds))
+                self._log.info('Too many API requests. Retrying after {} \
```

Comment:
Maybe swap these two (and reword the message accordingly), such that it's clear what's happening even if the delay becomes quite long for some reason.

Reply:
Done!